src/artm/messages.proto

```diff
@@ -387,6 +389,21 @@ message TopicKernelScore {
   optional StringArray topic_name = 10;
 }
 
+// Represents a configuration of a topic part in nwt score
+message TopicPartInNwtScoreConfig {
+  optional float eps = 1 [default = 1e-37];
+  optional string class_id = 2 [default = "@default_class"];
+  repeated string topic_name = 3;
+  repeated string topic_name_to_score = 4;
```

Comment:
I think it is getting a little bit too complicated with two lists (`topic_name` and `topic_name_to_score`). Maybe it would be sufficient to only keep one list (`topic_name`), and calculate the cumulative mass across `topic_name` devided by cumulative mass of **all topics**? In addition the score should output a vector of cumulative mass for each topic. This will enable the user to calculate the ratio between any two sets of topics.

The resulting messages would then look as follows:
```
message TopicMassPhiScoreConfig {
  optional float eps = 1 [default = 1e-37];
  optional string class_id = 2 [default = "@default_class"];
  repeated string topic_name = 3;
}

message TopicMassPhiScore {
  optional double value = 1;
  repeated string topic_name = 2; 
  repeated double topic_mass = 3;
}
```

The `TopicMassPhiScore.topic_mass ` will give the cumulative score for each topic. The names of each topic will be in `TopicMassPhiScore.topic_name`.  Note that the field `TopicMassPhiScore.topic_name` will contains all topics, e.g. it will not match the content of the input field `TopicMassPhiScoreConfig.topic_name`. I think this is OK.